raiden/ui/cli.py

```diff
@@ -236,232 +248,238 @@ def wait_for_sync(blockchain_service, url, tolerance, sleep):
-        is_flag=True
-    ),
-    click.option(
-        '--max-unresponsive-time',
-        help=(
-            'Max time in seconds for which an address can send no packets and '
-            'still be considered healthy.'
+        option(
+            '--registry-contract-address',
+            help='hex encoded address of the registry contract.',
+            default=ROPSTEN_REGISTRY_ADDRESS,  # testnet default
+            type=ADDRESS_TYPE,
+            show_default=True,
         ),
-        default=30,
-        type=int,
-        show_default=True,
-    ),
-    click.option(
-        '--send-ping-time',
-        help=(
-            'Time in seconds after which if we have received no message from a '
-            'node we have a connection with, we are going to send a PING message'
+        option(
+            '--discovery-contract-address',
+            help='hex encoded address of the discovery contract.',
+            default=ROPSTEN_DISCOVERY_ADDRESS,  # testnet default
+            type=ADDRESS_TYPE,
+            show_default=True,
         ),
-        default=60,
-        type=int,
-        show_default=True,
-    ),
-    click.option(
-        '--console',
-        help='Start the interactive raiden console',
-        is_flag=True
-    ),
-    click.option(
-        '--rpc/--no-rpc',
-        help='Start with or without the RPC server.',
-        default=True,
-        show_default=True,
-    ),
-    click.option(
-        '--sync-check/--no-sync-check',
-        help='Checks if the ethereum node is synchronized against etherscan.',
-        default=True,
-        show_default=True,
-    ),
-    click.option(
-        '--api-address',
-        help='"host:port" for the RPC server to listen on.',
-        default='127.0.0.1:5001',
-        type=str,
-        show_default=True,
-    ),
-    click.option(
-        '--datadir',
-        help='Directory for storing raiden data.',
-        default=os.path.join(os.path.expanduser('~'), '.raiden'),
-        type=click.Path(
-            exists=False,
-            dir_okay=True,
-            file_okay=False,
-            writable=True,
-            resolve_path=True,
-            allow_dash=False,
+        option(
+            '--console',
+            help='Start the interactive raiden console',
+            is_flag=True
         ),
-        show_default=True,
-    ),
-    click.option(
-        '--password-file',
-        help='Text file containing the password for the provided account',
-        default=None,
-        type=click.File(lazy=True),
-        show_default=True,
-    ),
-    click.option(
-        '--web-ui/--no-web-ui',
-        help=(
-            'Start with or without the web interface. Requires --rpc. '
-            'It will be accessible at http://<api-address>. '
+        option(
+            '--transport',
+            help='Transport system to use. Matrix is experimental.',
+            type=click.Choice(['udp', 'matrix']),
+            default='udp',
```

Comment:
For red eyes I suppose the defaults would be swapped

Reply:
Correct